aslp/aslp_client_server_ocaml.nix

```diff
@@ -6,24 +6,25 @@
 , mtime
 , aslp
 , aches
+, aslp_offline
 }:
 
 buildDunePackage {
   pname = "aslp_client_server_ocaml";
-  version = "0.1.1-unstable-2025-02-04";
+  version = "0.1.2-unstable-2025-02-05";
```

Comment:
normally i like to put version bumps in their own commits with clear names, but it's fine this time

Reply:
can the "add aslp_offline to its dependees" commit list all the packages it affects in the commit title